tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,300 @@
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom post-processing  | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| with CompositorEffects  |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Antialiasing
+~~~~~~~~~~~~
+
+See :ref:`doc_3d_antialiasing` for more information.
+
++-------------------+--------------------------+--------------------------+--------------------------+
+| Feature           | Compatibility            | Mobile                   | Forward+                 |
++===================+==========================+==========================+==========================+
+| MSAA              | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| TAA               | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| FSR2              | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| FXAA              | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| SSAA              | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------+--------------------------+--------------------------+--------------------------+
+| Screen-space      | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| roughness limiter |                          |                          |                          |
++-------------------+--------------------------+--------------------------+--------------------------+
+
+StandardMaterial features
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See :ref:`doc_standard_material_3d` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| Sub-surface scattering  | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
```

Comment:
I focused on only adding features that *varied* between the renderers, but that means that some tables such as this one are very short. I could add in some more features, like `triplanar` or `refraction` that are supported by all renderers, too.

Reply:
Left a comment for this on the other one.